cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -2,70 +2,74 @@
-                {% endif %}
+                    <span class="current">
+                        Page {{ features.number }} of {{ features.paginator.num_pages }}
+                    </span>
 
-                <span class="current">
-                    Page {{ features.number }} of {{ features.paginator.num_pages }}
+                    {% if features.has_next %}
+                        <a href="?page={{ features.next_page_number }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.next_page_number }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-target="#features-table">next</a>
+                        <a href="?page={{ features.paginator.num_pages }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.paginator.num_pages }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-target="#features-table">last »</a>
+                    {% endif %}
                 </span>
-
-                {% if features.has_next %}
-                    <a href="?page={{ features.next_page_number }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.next_page_number }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-target="#features-table">next</a>
-                    <a href="?page={{ features.paginator.num_pages }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-get="{% url 'search:dna_feature_loc' loc.chr loc.start loc.end %}?page={{ features.paginator.num_pages }}&dist={{ dist }}{% for type in feature_types %}&feature_type={{ type }}{% endfor %}&assembly={{ assembly|default:'GRCh38' }}" hx-target="#features-table">last »</a>
-                {% endif %}
-            </span>
-        </div>
-
+            </div>
     </div>
+        {% else %}
+            <div class="table-title" style="display: flex; align-items: center; justify-content: center">
```

Comment:
Please don't use inline style tags. Either use the relevant tailwind classes or, if you find you're using the same group of styles over and over, put those styles in a new CSS class and use that class.